tests/unit/test_utils.py

```diff
@@ -0,0 +1,20 @@
+import numpy as np
+import pandas as pd
+
+from pseudopeople.schema_entities import DtypeNames
+from pseudopeople.utilities import cleanse_integer_columns
+
+
+def test_clenase_object_columns():
+    # This tests that object columns return only strings.
+    # This is to handle dtype issues we were having with int/float/strings in
+    # age, wages, and po box columns.
+    s = pd.Series([np.nan, 1, "2.0", 3.0, 4.0, "5.0", np.nan])
```

Comment:
What I alluded to above - are there guarantees that these floats will always have 1 decimal? Might be worth making it more generic, i.e. this would fail w/ "6.00"

Reply:
I guess thats a good point. All these issues seem to be caused by ints changing to floats because of nans so there currently is always only one decimal point but it wouldnt hurt to make it more generic.